wiki/Registration/en.md

```diff
@@ -1,26 +1,21 @@
----
-outdated: true
----
-
 # Registration
 
 *Warning: Having more than one osu! user account at any time is an infringement against the [osu! rules](/wiki/Rules)!*
 
-1. After [installing](/wiki/Installation) osu! and running it, osu! will prompt you to sign in or register. Assuming that you're here because you need an account, click `Create an account` and keep osu! opened. This should open the [Create Account](https://osu.ppy.sh/p/register) web page.
+1. After [installing](/wiki/Installation) osu! and running it, osu! will prompt you to sign in or register. Assuming that you're here because you need an account, click `Create an account`. Enter your username (if it's not taken), your email, and your password. Then click `Create Account`.
 
 ![The registration form from the website](img/register.jpg "The registration form")
 
 2. Fill out all of the fields in the form.
    - For the username field, there is a maximum of 15 characters; of which can only be alphanumeric characters (`a-z`, `A-Z`, `0-9`), underscores (`_`), square brackets, (`[` and `]`), dashes (`-`), and spaces (` `).
    - For the email address field, use your email. This is used to reset your password and to send you verification codes when needed, so don't use a throw away email.
    - For the password field, use a password that is at least 8 characters long.
-3. Once this is completed, press `Continue` then go back to osu!
-4. Sign in to complete registration.
+3. Once this is completed, press `Create Account`. You will be automatically signed in.
 
 ## What's next?
 
 Don't forget to read the [rules](/wiki/Rules) very carefully!
 
-Once that is done, you are now ready to start your rhythm adventure! You can [add beatmaps](/wiki/Installation#adding-beatmapsets) to play, [make a skin](/wiki/Skinning), or [make a beatmap](/wiki/Beatmapping). You could also stop by and say, "Hi," in the [Introductions subforum](https://osu.ppy.sh/community/forums/8).
+Once that is done, you are now ready to start your rhythm adventure! You can [add beatmaps](/wiki/Installation#adding-beatmapsets) to play, [make a skin](/wiki/Skinning), or [make a beatmap](/wiki/Beatmapping). You could also stop by and say, "Hi" in the [Introductions subforum](https://osu.ppy.sh/community/forums/8).
```

Comment:
I guess the comma was used incorrectly, can probably remove the other one before it too.

```suggestion
Once that is done, you are now ready to start your rhythm adventure! You can [add beatmaps](/wiki/Installation#adding-beatmapsets) to play, [make a skin](/wiki/Skinning), or [make a beatmap](/wiki/Beatmapping). You could also stop by and say "Hi" in the [Introductions subforum](https://osu.ppy.sh/community/forums/8).
```